frontend/src/pages/tracker-report.page.tsx

```diff
@@ -181,6 +181,21 @@ const TrackerReport: NextPage = () => {
             <div className={styles.entries}>
               <FaqAccordionItem
                 entries={[
+                  {
+                    q: l10n.getString("faq-question-define-tracker-question"),
+                    a: (
+                      <p>
+                        {l10n.getString(
+                          "faq-question-define-tracker-answer-partone"
+                        )}
+                        <br />
+                        <br />
+                        {l10n.getString(
+                          "faq-question-define-tracker-answer-parttwo"
+                        )}
+                      </p>
```

Comment:
Nit, but aren't these just two paragraphs?

```suggestion
                      <>
                        <p>
                          {l10n.getString(
                            "faq-question-define-tracker-answer-partone"
                          )}
                        </p>
                        <p>
                          {l10n.getString(
                            "faq-question-define-tracker-answer-parttwo"
                          )}
                        </p>
                      </>
```